Adapter_Revit_UI/Pull/Push.cs

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+
+using BH.oM.Data.Requests;
+
+namespace BH.UI.Revit.Adapter
+{
+    public partial class RevitUIAdapter
+    {
+        /***************************************************/
+        /**** Protected Methods                         ****/
+        /***************************************************/
+
+        public override IEnumerable<object> Pull(IRequest request, Dictionary<string, object> config = null)
+        {
+            return Read(request);
+        }
```

Comment:
This Pull override should not be needed as the base adapter works the same:
Please see https://github.com/BHoM/BHoM_Adapter/blob/master/BHoM_Adapter/AdapterActions/Pull.cs
Please have a look and if confirmed, delete the override. 

Also, I see the name of the file here is `Push.cs`, which is incorrect. Please always put AdapterActions overrides like Push, Pull etc in single files (one file per action overridden) named after them.